com.unity.render-pipelines.universal/Runtime/UniversalRenderPipelineCore.cs

```diff
@@ -552,32 +555,58 @@ void SortCameras(Camera[] cameras)
             return desc;
         }
 
-        static Lightmapping.RequestLightsDelegate lightsDelegate = (Light[] requests, NativeArray<LightDataGI> lightsOutput) =>
+        private static Lightmapping.RequestLightsDelegate lightsDelegate = (Light[] requests, NativeArray<LightDataGI> lightsOutput) =>
         {
             LightDataGI lightData = new LightDataGI();
 #if UNITY_EDITOR
             // Always extract lights in the Editor.
             for (int i = 0; i < requests.Length; i++)
             {
                 Light light = requests[i];
+
+                var additionalLightData = light.GetComponent<UniversalAdditionalLightData>();
+                if (additionalLightData == null)
+                {
+                    additionalLightData = ComponentSingleton<UniversalAdditionalLightData>.instance;
```

Comment:
It's not adding a component (or at least that wasn't the intent), but get's reference to a default.
I can add couple ifs instead of passing a "default".
What do you think?

Reply:
I saw this used somewhere else, so I tried to do it the same way.

Reply:
As discussed in thread we will do this in a future PR. The work is not relevant to this PR and should not depend on each other. It is ok to move forward with this.